Python/sort/comb_sort.py

```diff
@@ -0,0 +1,53 @@
+"""
+CombSort Algorithm Implementation in Python 3
+
+Purpose:
+Comb sort is a relatively simple sorting algorithm.
+Comb sort improves on bubble sort algorithm.
+In bubble sort, distance (or gap) between two compared elements is always one.
+Comb sort improvement is that gap can be much more than 1, in order to prevent
+slowing down by small values at the end of a list.
+
+Time Complexity :
+Worst case complexity is O(n^2)
+Best Case complexity is O(n)
```

Comment:
@mehabhalodiya I think the best time complexity will be `O(n log n)` and that will occur when all elements are already sorted or nearly sorted. In that case, the loop with gap=1 will be run only once (as the others).  It will follow the sequence of generalized harmonic numbers. Also, add about space complexity.